test_scripts/API/VehicleData/UnsubscribeVehicleData/001_Success_flow.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed by Policies
+--
```

Comment:
@AByzhynar, add link to the requirement when they will be available.

Reply:
@AStasiuk Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45